src/components/pages/customers/reviews.tsx

```
@@ -0,0 +1,70 @@
+  tag,
+}: ICustomerTweetData) => {
+  return (
+    <div className="relative h-full w-full md:max-w-[384px]">
+      <Link
+        className="relative z-10 flex h-full flex-col items-start rounded-xl border border-[rgba(51,51,71,0.60)] bg-[#111018] px-5 py-[18px] text-start transition-colors duration-300 hover:bg-[#15141D] md:px-6 md:py-5"
+        href={tweetLink}
+        variant="white"
+      >
+        <p
+          className="[&>span]:text-primary-1 mb-5 line-clamp-5 text-[15px] leading-snug font-[350] xl:text-base"
+          dangerouslySetInnerHTML={{ __html: text }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)